ada/language/ast.py

```diff
@@ -6414,6 +6414,25 @@ class DefiningName(Name):
         doc="Returns this DefiningName's basic declaration"
     )
 
+    @langkit_property(public=False, return_type=AdaNode.entity.array)
+    def find_all_refs_in(x=AdaNode.entity):
```

Comment:
Even though this properly is an implementation helper, describing how it’s supposed to be used and what is does would not hurt :-)